src/main/webapp/index.html

```diff
@@ -1,18 +1,18 @@
  <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Welcome Page</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Cinzel">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lora">
-  </head>
- <body>
-    <div id="content">
-        <br></br><br></br><br></br>
-        <div class="title">houseHOLD</div>
-        <h3 class="body-font">Please log in here:</h3>
-    </div>
+    <head>
+        <meta charset="UTF-8">
+        <title>Welcome Page</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Cinzel">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lora">
+    </head>
+    <body style="background-color: rgb(205, 236, 232);">
```

Comment:
Would it be better to keep this in the stylesheet? Presumably you want the same background color on every page.